internal/config/config.go

```diff
@@ -282,6 +284,11 @@ func ParseDSN(dsn string) (UserConfig, error) {
 		ucfg.Location, err = time.LoadLocation(timezone)
 	}
 
+	if params.Has("stagingAllowedLocalPath") {
```

Comment:
I think it would be cleaner to do:
```
if paths, ok := params.extract("stagingAllowedLocalPath"); ok {
    ucfg.StagingAllowedLocalPath = catalog
}
```